src/components/SelectorVisualMediaGrid/SelectorVisualMediaGridPresentation.js

```diff
@@ -36,7 +35,7 @@ function SelectorVisualMediaGridPresentation({
                   const mediaSrc = getMediaPath({
                     type: IMAGE,
                     mediaObject,
-                    size: THUMBNAIL,
```

Comment:
Is this change of thumbnails to small images to fix the broken video thumbnails?